sections/exchange/TradeCard/Charts/common/ChartTypeIconsToggle.tsx

```diff
@@ -46,9 +45,19 @@ const ChartTypeToggle: FC<ChartTypeToggleProps> = ({
 					}}
 				>
 					{chartType === ChartType.AREA ? (
-						<StyledSvg src={AreaIcon} isActive={chartType === selectedChartType} />
+						<AreaIcon
+							filter={
+								chartType === selectedChartType &&
+								'invert(58%) sepia(9%) saturate(1019%) hue-rotate(203deg) brightness(95%) contrast(88%)'
+							}
+						/>
 					) : (
-						<StyledSvg src={CandlesticksIcon} isActive={chartType === selectedChartType} />
+						<CandlesticksIcon
+							filter={
+								chartType === selectedChartType &&
+								'invert(58%) sepia(9%) saturate(1019%) hue-rotate(203deg) brightness(95%) contrast(88%)'
+							}
+						/>
```

Comment:
I think you can put the filter value in a variable and pass it as props to remove the need for duplication.

Reply:
@koredefashokun You leveled up my TS skills :thumbsup: